### ISSUE TITLE
Migrations issue running Razor sample for first time

### ISSUE THREAD

**Issue by timgabrhel** [PiranhaCMS/piranha.core#1289]
Trying to get started exploring the new Piranha Core, but running in to an issue on first time setup. I've done a clone on the full project source, currently on Master.

- Using the Razor example project.
1. I ran the default SqlLite template just fine. Was able to explore the site and manager.
2. Add an appsettings.json and set my connection string.
3. Point both EF & Identity to my SQL server DB (SQL Azure database)
4. Run the project
5. See error referring to a newer `LogoId` column on the Site.
![image](https://user-images.githubusercontent.com/791022/89313378-0a755700-d63e-11ea-8ebd-6c7d25cf7f57.png)
6. It appears this is a newer column as found in this migration `20200625060542_AddSiteLogo.cs`
![image](https://user-images.githubusercontent.com/791022/89313480-25e06200-d63e-11ea-8ed1-4cbaf2d08a9f.png)
7. I'm not seeing the AddSiteLogo migration in the EFMigrationHistory.
![image](https://user-images.githubusercontent.com/791022/89313533-3a245f00-d63e-11ea-9bd1-b3cda91b5dcd.png)

I'm not sure if the migration is failing, or if it's simply not executing. I'm not as familiar with EF migrations so some guidance is appreciated.

**Comment by tidyui** [comment]
Hi there! Have you changed packages to target SQLServer? You need to change both the data package as well as the Identity package! See this page for reference:

https://piranhacms.org/docs/architecture/databases/sql-server

Best regards

**Comment by timgabrhel** [comment]
@tidyui Yes, sorry, I forgot to mention. I installed both packages as the necessary types for SQL EF and SQL Identity aren't already present. I did not _uninstall_ any packages, only added the two referenced on this page.

**Comment by tidyui** [comment]
@timgabrhel I actually also missed that you were running on master and not the current packages 😁 Master is our development branch and might not be stable. The SiteLogo field is new for 8.4 and we probably haven’t had time to create the migration for SQLServer yet!

If you’re happy with the 8.3 functionality you can find the latest stable release in the version branch!

Best regards

Håkan

**Comment by tidyui** [comment]
@filipjansson Can you take a look at this when you get the time. It looks like you have already created the migration for `SQLServer` but it doesn't appear to be executing correctly.

**Comment by timgabrhel** [comment]
@tidyui Might I suggest some guidance of this note, perhaps under this section.
https://github.com/PiranhaCMS/piranha.core#get-the-latest-source-code-and-get-going

**Comment by tidyui** [comment]
Yes good idea! I’ll rename this issue and move it to the documentation repo!
